projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -81,8 +86,12 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     protected readonly groupId?: string,
     private readonly chartSyncService?: ChartSyncService<TData>
   ) {
-    this.chartSyncService?.locationChangeSubject.subscribe(data => {
-      this.showTooltip(data);
+    this.chartSyncService?.locationChange$.subscribe(data => {
```

Comment:
Should only be subscribed to if we have an ID.

Reply:
> @aaron-steinfeld groupid should be sent from the primary chart along with the current location. right?. Here we are sending from the nonprimary charts. How can we compare whether these are the same or not?

@jyothishjose6190 there is no "primary chart" unless you mean the one being hovered. The one being hovered would broadcast, across the chart sync service that its groupId is being hovered at some location. Here, we're listening - which we only need to do if we have a group ID, because we only want to listen for events relevant to our group ID.